tests/lax_control_flow_test.py

```diff
@@ -2161,7 +2242,7 @@ def dummy_root_usage(x):
   def test_custom_linear_solve(self, symmetric):
 
     def explicit_jacobian_solve(matvec, b):
-      return lax.stop_gradient(jnp.linalg.solve(jax.jacobian(matvec)(b), b))
+      return jnp.linalg.solve(jax.jacobian(matvec)(b), b)
```

Comment:
I do actually like keeping stop_gradient here (and in the other `custom_linear_solve` tests), becauses it ensures that we're testing the custom gradient instead of the gradient that is defiend by `jnp.linalg.solve`.

Reply:
Will add back.